resources/js/components/Home/RegisterForm.vue

```diff
@@ -0,0 +1,153 @@
+        >
+      </div>
+    </div>
+
+    <div>
+      <div class="flex items-center justify-between">
+        <label for="password" class="block text-sm font-medium leading-6 text-gray-900">Hasło</label>
+      </div>
+      <div class="mt-2">
+        <input id="password" required name="password" type="password" autocomplete="current-password"
+               class="duration-200 ring-inset outline-none focus:ring focus:ring-primary bg-white/30 rounded-lg w-full p-3 text-gray-900 ring-2 ring-primary/30 placeholder:text-gray-400"
+        >
+      </div>
+    </div>
+
+    <div>
+      <div class="flex items-center justify-between">
+        <label for="search-schools" class="block text-sm font-medium leading-6 text-gray-900">Szukaj szkół</label>
+      </div>
+      <div :class="{'scale-y-100 max-h-80':isSearchFocused}" class="overflow-hidden max-h-12 flex flex-col mt-2 bg-white/30 placeholder:text-gray-400 rounded-[.5rem] ring-2 ring-primary/30 ring-inset">
+        <input 
+          v-model="schoolsSearchBar"
+          class="outline-none focus:duration-200 focus:ring-inset focus:ring focus:ring-primary rounded-[.5rem] p-3 bg-transparent w-full text-gray-900" required name="search-schools"
+          :class="{'cursor-pointer' : !isSearchFocused}"
+          type="text"
+          @focus="isSearchFocused = true; schoolsSearchBar=''"
+          @blur="isSearchFocused = false; schoolsSearchBar=selectedSchool"
+        >
+        <Transition>
+          <div v-show="true" class="m-0.5 mt-0 py-2 overflow-auto">
+            <div v-if="filteredSchools.length>0">
+              <span v-for="obj in filteredSchools"
+                    :key="obj.id"
+                    class="cursor-pointer block px-4 py-2 hover:bg-primary/10 text-[0.9rem]"
+                    @mousedown="onListOptionClick(obj)"
+              >{{ obj.data }}</span>
+            </div>
+            <span v-else class="block px-4 py-2 text-sm">
+              Nie znaleziono szkoły
+            </span>
+          </div>
+        </Transition>
+      </div>
+    </div>
+
+    <div class="mx-2 mt-4 flex flex-row items-center gap-4">
+      <Checkbox />
+      <p class="w-fit text-sm text-gray-500">
+        Akceptując ten warunek, zgadzasz się na
+        <a href="#" class="font-semibold leading-6 text-primary hover:primary">warunki usługi</a> oraz
+        <a href="#" class="font-semibold leading-6 text-primary hover:primary">regulamin</a>
+      </p>
+    </div>
+
+    <div>
+      <button type="submit"
+              class="
+              rounded-lg text-md flex w-full justify-center bg-primary p-3 font-bold text-white transition hover:bg-primary-950
+              focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-primary"
+      >
+        Kontynuuj
```

Comment:
Why not `Zarejestruj się` or something like this

Reply:
Because the upper registration tab already says `Rejestracja`, I find `Zarejestruj się` button name tautologous. I do not insist on keeping the current name though, in fact initially I named it the same way as you did.

Reply:
I will stick to "Zarejestruj się`

Reply:
```suggestion
        Zarejestruj się
```